cpp/velox/substrait/SubstraitToVeloxPlan.cc

```diff
@@ -1292,18 +1292,20 @@ core::PlanNodePtr SubstraitToVeloxPlanConverter::toVeloxPlan(const ::substrait::
 
   // Velox requires Filter Pushdown must being enabled.
   bool filterPushdownEnabled = true;
+  auto names = colNameList;
```

Comment:
`readRel.base_schema` are output schema and are not consistent with parquet file schema, so it seems that `readRel.base_schema` cannot be set as hive fileSchema.

Reply:
Since we map by filed name instead of indices, this change is acceptable.

https://github.com/apache/incubator-gluten/blob/e8b723791086fd520e1c97f6da4f969783da2215/cpp/velox/compute/VeloxBackend.cc#L313-L315